src/azure-cli/azure/cli/command_modules/util/_help.py

```diff
@@ -22,6 +22,12 @@
 
 
     For passing JSON in PowerShell, see https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md
+
+
+    [WARNING] Depending on the REST API you call,
+    the output may include secrets that you must protect. Be sure that you do not include these secrets in your
+    source control. Also verify that no secrets are present in the logs of your command or script.
```

Comment:
```suggestion
    source control. Also verify that no secrets are present in the output of your command or script that may be stored in various logs files.
```

Reply:
I will ignore this comment as it is a draft version.